modules/microshift-mDNS.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_content-type: CONCEPT
+[id="microshift-mDNS_{context}"]
+== mDNS
```

Comment:
If this is the first mention of mDNS, it shouldn't be abbreviated, per IBMSG guidance on abbreviations.  Multicast DNS (mDNS)